iroh-blobs/Cargo.toml

```diff
@@ -72,7 +72,7 @@ package = "iroh-quinn"
 version = "0.10"
 
 [features]
-default = ["fs-store"]
+default = ["fs-store", "metrics"]
```

Comment:
so this means metrics collection is on by default in blobs now, or just that the feature is available?

Reply:
Reworked so that dep is always available but metrics collection is not on by default.